src/Raven.Client/Documents/Session/Operations/LoadOperation.cs

```diff
@@ -12,11 +13,10 @@ internal class LoadOperation
         private readonly InMemoryDocumentSessionOperations _session;
         private static readonly Logger Logger = LoggingSource.Instance.GetLogger<LoadOperation>("Client");
 
-        private string[] _ids;
+        private readonly List<string> _ids = new List<string>();
```

Comment:
better to use array only, then you will not have to do `.ToArray()` in `CreateRequest`